src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,196 @@
+package mate.jdbc.dao;
```

Comment:
You have to create the package impl for dao implementations, the same for services